tests/unit_node/process_test.ts

```diff
@@ -226,6 +226,7 @@ Deno.test({
   },
 });
 
+// Only supported on Windows (but won't throw anywhere)
```

Comment:
Updated these two tests to correct expectations (and ensure that we do leak detection for this on all platforms)